docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -65,9 +65,58 @@ boosted tree regression model which combines multiple weak models into a
 composite one. We use decision trees to learn to predict the probability that 
 a data point belongs to a certain class.
 
+
+[discrete]
+[[dfa-classification-interpret]]
+==== Interpreting {classification} results
+
+The following sections help you understand and interpret the results of a 
+{classanalysis}.
+
+[discrete]
+[[dfa-classification-class-probability]]
+===== `class_probability`
+
+The value of `class_probability` shows how likely it is that a given datapoint 
+belongs to a certain class. It is a value between 0 and 1. The higher the 
+number, the higher the probability that the data point belongs to the named 
+class. For further details, see the
+{ml-docs}/flightdata-classification.html#flightdata-classification-results[Viewing {classification} results]
+section in the {classification} example.
+
+
+[discrete]
+[[dfa-classification-class-score]]
+===== `class_score`
```

Comment:
It is unclear where to find this score.  I don't see it in the results example in http://stack-docs_885.docs-preview.app.elstc.co/guide/en/machine-learning/master/flightdata-classification.html#flightdata-classification-results

Reply:
AFAIK, `class_score` has become exposed recently. I'll update the API examples in a separate PR.